composer.json

```diff
@@ -4,19 +4,23 @@
     "keywords": ["doctrine", "aes256", "encrypt", "decrypt"],
     "license": "MIT",
     "description": "Package encrypts and decrypts Doctrine fields through life cycle events.",
-
     "require": {
-        "php": ">=5.3.2",
-        "doctrine/orm": "~2.0",
-        "ext-mcrypt": "*"
+        "php": ">=7.1",
```

Comment:
Can confirm that I considered it and and decided to totally ignore your warnings.